task.go

```diff
@@ -140,6 +140,10 @@ type TaskInfo struct {
 	RootFS []mount.Mount
 	// Options hold runtime specific settings for task creation
 	Options interface{}
+	// RuntimePath is an absolute path that can be used to overwrite path
+	// to a shim runtime binary.
+	RuntimePath string
+
```

Comment:
`runtime` field is now unused? Do we need to keep both?

Reply:
The runtime field will be set to `container.Runtime.Name`, which will be used for TaskOpts such as `WithNoPivotRoot` to determine the runtime
https://github.com/containerd/containerd/blob/29e10a192a676628080dd79b9500e609df3b470f/task_opts_unix.go#L55-L60

Reply:
Could you add comment lines to explain that?

Reply:
Added some comments to the runtime field

Reply:
The problem is that the client doesn't use the services/tasks/v1.CreateTaskRequest.RuntimePath.

Container info has field named by `Runtime.Name`. It's consistency with keeping both. `RuntimePath` is more like override option.